newIDE/app/src/UI/Search/UseSearchStructuredItem.js

```diff
@@ -205,6 +205,7 @@ export const useSearchStructuredItem = <
             { name: 'name', weight: 2 },
             { name: 'fullName', weight: 5 },
             { name: 'shortDescription', weight: 1 },
+            { name: 'tags', weight: 4 },
```

Comment:
What do you think of the weight?

Reply:
tags are arrays, can you check Fuse.js is working properly with these?

Reply:
It handles arrays according to the documentation:
https://fusejs.io/api/options.html#keys